.github/workflows/build_and_release.yml

```diff
@@ -84,13 +84,21 @@ jobs:
         if: matrix.target_os == 'linux' && matrix.target_arch == 'aarch64'
         run: |
           sudo apt-get update
-          sudo apt-get install build-essential libssl-dev pkg-config cmake protobuf-compiler -y
+          sudo apt-get install build-essential libssl-dev pkg-config cmake protobuf-compiler unixodbc unixodbc-dev -y
+
+      # The x86_64 runner does not unixodbc pre-installed
+      - name: Install missing tools (Linux x86_64)
```

Comment:
Still waiting on https://github.com/spiceai/spiceai/actions/runs/10035934670 to complete to validate this fixes the issue